wordpress-mobile/WordPress-iOS
# Reader: Remove obsolete code interfering with content insets.

Fixes #16623

This PR removes some obsolete code from an earlier version of the reader that was modifying the reader detail's `scrollView`'s `contentInsets` in a way that cropped part of the header view.  The code is no longer needed as featured images are handled differently in the current iteration of the reader.  In the issue description it's reported that posts with featured images weren't affected, however it looks like the cropping was just less obvious.

@fluiddot would you be up for a quick review since you're familiar with the issue? 

To test:
Test on both an iPhone and an iPad form factor.  
Test with a short post, a long post, a pos with a featured image, and a post without a featured image.
- View the post in the reader.
- Rotate the device and confirm that content is no longer clipped at the top of the post. 
- Rotate back and confirm that content is still not clipped.

## Regression Notes
1. Potential unintended areas of impact
Reader detail presentation

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Smoke tested many posts in different orientations.

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
